domains/rohitsonawane.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "rohitsinhsonawane",
+        "email": "rohitsinh.sonawane@gmail.com"
+    },
+    "record": {
+        "CNAME": "rohitsinhsonawane.github.io"
```

website is incomplete
```suggestion
    "records": {
        "CNAME": "rohitsinhsonawane.github.io"
```